### DISCUSSION TITLE
win11 pro - Flastool 0.9.36

### DISCUSSION THREAD

**Discussion by jpg-93** [Androxyde/Flashtool#215] [General]
Hi all
No way to run 0.9.36 with win11 for my sony xperia z5 compact. Only work 0.9.28.
Am i alone ?

**Comment by Chris8top8he8r** [comment]
I'm running windows 11 with 0.9.36 with Xperia XA2 and LineageOS 21 (Android 14).  I just rooted phone with Magisk and it checks as rooted, but cannot get FT to get the root permission I'm granting.  The phone toasts that "Shell" has been granted root access, but the 0.9.36  app always says root access denied.  In Magisk settings I said grant app and ADB, and same failure.  Then I added Automatic Response: Grant, but same error.  All this because I want to relock my bootloader.  Sony support rep said there is a 3rd party app to relock bootloader with LineageOS and I figured FT is it.  Sony rep said he couldn't name the 3rd party app.  But at this point I can't access FT menu under "BLU" until I get root access solved.  Any one know if this should work?
